pkgs/desktops/xfce4-14/thunar/default.nix

```diff
@@ -1,5 +1,5 @@
 { mkXfceDerivation, docbook_xsl, exo, gdk-pixbuf, gtk3, libgudev ? null
-, libnotify ? null, libX11, libxfce4ui, libxfce4util, libxslt, xfconf, gobject-introspection }:
+, libnotify ? null, libX11, libxfce4ui, libxfce4util, libxslt, xfconf, gobject-introspection, gvfs }:
```

Comment:
those `? null` across `xfce4-14` could be cleaned up, I see no purpose to have fallback value for params which are always filled by `callPackage`

Reply:
Agreed, done in https://github.com/NixOS/nixpkgs/pull/66519/commits/019d33d5f02d560fd19acd8bda9ee6a954356083